core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/FromClause.kt

```diff
@@ -18,7 +18,8 @@ class FromClause(private val fromable: Fromable, private val parentClause: ISele
         }
         return DopeQuery(
             queryString = formatToQueryStringWithSymbol(parentDopeQuery.queryString, "FROM", fromableDopeQuery.queryString),
-            parameters = fromableDopeQuery.parameters + parentDopeQuery.parameters,
+            parameters = parentDopeQuery.parameters + fromableDopeQuery.parameters,
+            positionalParameters = parentDopeQuery.positionalParameters + fromableDopeQuery.positionalParameters,
```

Comment:
Can we merge positionalParamters like that? Let's assume parentDopeQuery and fromableDopeQuery have both two positionalParameters. If we merge it like that, then parentDopeQuery.queryString has $1 and $2 and fromableDopeQuery.queryString has $3 and $4 is this always guaranteed?

Reply:
With the new merge functionality the order is guaranteed as long as the parent DopeQuery calls the merge function.